lib/http_request_helper.rb

```diff
@@ -0,0 +1,32 @@
+module HTTPRequestHelper
```

Comment:
What is the purpose of this module? Can it be implemented as a normal class?

Reply:
This module totally is a helper for simplifying http-requests. Moved it to /helpers/ anyway.